packages/nitro-protocol/docs/state-channels/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+id: overview
+title: Overview
+---
+Welcome to the nitro-protocol state channels documentation site. In this introductory pages, you will find a high level description of state channels; their use-cases and the challenges they represent; our solution to those challenges. You will also find a quick start guide to help you nitro-charge / channelize your DApp using our framework.
+
+Elsewhere on the site you will find API documentation for a reference implementation of nitro that is gas optimized and tested.
+
+## What is a State Channel?
+
+State channels are a technique improving the scalability of a blockchain, and for increasing throughput for some of its Decentralized Applications (DApps).
```

Comment:
This reads a bit odd to me. Maybe it should be something like:
"State channels are a technique `for` improving the..."